app/serializers/account_serializer.rb

```diff
@@ -129,12 +129,14 @@ class AccountSerializer < BaseSerializer
   meta do
     ed25519_key = Base64.strict_encode64(@object.ed25519_public_key)
     rsa2048_key = Base64.strict_encode64(@object.public_key)
+    ecdsa_key   = Base64.strict_encode64(@object.ecdsa_public_key) if @object.ecdsa_public_key?
```

Comment:
This guard does nothing. Instead, we should split the PR up into separate deployments:

1. Add ECDSA keypair columns, and default ECDSA Keygen for new accounts.
2. Backfill ECDSA keypairs for old accounts.
3. Deploy ECDSA signature schemes.

This will de-risk the deployment, namely reducing issues due to account caching.